mantidproject/mantid
# Fix detector rotations in TOFTOF IDF

This PR fixes TOFTOF IDF to account for detector rotations. 

**To test:**

Run
```python
ws = LoadEmptyInstrument(InstrumentName='TOFTOF')
```
Use `Show Instrument` and `Show detectors` options to see the instrument geometry.  The detectors must be faced to the sample now. The distance from sample to detector must not change and should be `4` meters for all the detectors.

More details on calculations of the rotation axis and angles are in the [script](https://github.com/mganeva/scripts/blob/master/toftof/correctIDF.py).

Fixes #23071. 

Does this update require release notes?


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
